src/tenant/initialization.py

```diff
@@ -41,11 +41,16 @@ def create_superusers():
         password=settings.TENANT_DEFAULT_ADMIN_PASSWORD
     )
     # OWNER OF THE DECK
-    User.objects.create_superuser(
-        username=settings.TENANT_DEFAULT_OWNER_USERNAME,
-        email='owner@example.com',
-        password=settings.TENANT_DEFAULT_OWNER_PASSWORD,
-    )
+    # THIS IS HERE BECAUSE siteconfig.models.get_default_deck_owner() will run before this file (initialization.py)
+    # although siteconfig.models.get_default_deck_owner() will not run before this every time based on testing
+    # so this catches an already made owner and skips it
+    owner = User.objects.filter(username=settings.TENANT_DEFAULT_OWNER_USERNAME).first()
+    if not owner:
+        User.objects.create_superuser(
```

Comment:
Is there a reason we can't use get_or_create() here? https://docs.djangoproject.com/en/3.2/ref/models/querysets/#get-or-create
That's much cleaner!

Reply:
replaced